wilson/wcxf/__init__.py

```diff
@@ -14,9 +14,9 @@
 child_bases = glob.glob(os.path.join(_root, 'bases', 'child', '*.basis.json'))
 
 for eft in all_efts:
-    with open(eft, 'r') as f:
+    with open(eft) as f:
```

Comment:
Is this really a good idea, to be less specific in how you open a file? I see that pyupgrade removes 'U' and 'r', which are the default now. What if this changes in a future Python version? If I'm overlooking anything, feel free to rebase and merge :-).